static/sass/_pattern_blog-topic.scss

```diff
@@ -0,0 +1,12 @@
+@mixin blog-p-topic {
+  .p-blog-topic__strip {
+    background-color: $color-accent-background;
+  }
+
+  .p-blog-topic__icon {
```

Comment:
Can we call this class something reuseable please. This makes this pattern unusable in another area of the site